dp/skaffold.yaml

```diff
@@ -159,58 +161,51 @@ profiles:
         path: /build/artifacts/-
         value:
           image: controller
+          context: ..
           custom:
-            buildCommand: "tools/skaffold_hooks/hooks.sh build_controller"
+            buildCommand: "dp/tools/skaffold_hooks/hooks.sh build_controller"
             dependencies:
               paths:
-                - "../dp/**"
-                - "../lte/*"
-                - "../orc8r/*"
+                - nms
+                - dp/cloud/go/**
+                - lte/**
+                - orc8r/**
               ignore:
-                - "../dp/skaffold.yaml"
-                - "../dp/tools/**"
-                - "../dp/Makefile"
-                - "../dp/cloud/helm/**"
-                - "../lte/cloud/helm/lte-orc8r/charts/**"
-                - "../orc8r/cloud/helm/orc8r/charts/**"
-                - "../**/*.swp"
+                - dp/cloud/go/active_mode_controller/**
+                - lte/cloud/helm/lte-orc8r/charts/**
+                - orc8r/cloud/helm/orc8r/charts/**
+                - "**/*.swp"
       - op: add
         path: /build/artifacts/-
         value:
           image: orcnginx
+          context: ..
           custom:
-            buildCommand: "tools/skaffold_hooks/hooks.sh build_nginx"
+            buildCommand: "dp/tools/skaffold_hooks/hooks.sh build_nginx"
             dependencies:
               paths:
-                - "../dp/**"
```

Comment:
Also not sure about this, since the orc8r is a monolith and it contains also a dp service

Reply:
Here are dependencies for orcnginx artifact which is just reverse-proxy to obsidian, so dp dependency it's not necessary